build/tools/msvs/package.bat

```diff
@@ -86,23 +72,12 @@ rem Change to the directory containing wxwidgets.props in order to include it
 rem into the archive without any path.
 cd %packagePath%\%VCver%
 
-7z a -t7z wxMSW-%wxMAJOR_VERSION%.%wxMINOR_VERSION%.%wxRELEASE_NUMBER%_%VCver%_Dev.7z wxwidgets.props
 7z a -t7z wxMSW-%wxMAJOR_VERSION%.%wxMINOR_VERSION%.%wxRELEASE_NUMBER%_%VCver%_x64_Dev.7z wxwidgets.props
 
 del wxwidgets.props
 
-rem Change to the directory containing licence file in order to include it
```

Comment:
Has this block been removed intentionally? We still want to include the licence.